chain/beacon/node.go

```diff
@@ -251,7 +251,10 @@ func (h *Handler) run(startTime int64) {
 				// already. If that next beacon is created soon after, this
 				// channel will trigger again etc until we arrive at the correct
 				// round.
-				h.broadcastNextPartial(current, b)
+				go func(c roundInfo, latest *chain.Beacon) {
+					time.Sleep(h.conf.Group.CatchupPeriod)
```

Comment:
I'm not sure about putting that in the group file - why not simply as a build constant ?

Reply:
You need to use `h.conf.Clock.Sleep` to use the clock interface.